build fails; missing package react-router-redux

<!--- Provide a general summary of the issue in the title above -->

## Expected Behavior
<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->
npm run build:dev should succeed

## Current Behavior
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from the current behavior -->

Following error occurs after clean `npm install` followed by `npm run build:dev`
```
ERROR in ./src/redux/actions/auth.js
Module not found: Error: Can't resolve 'react-router-redux' in '/home/mjl/Projects/riff/riff-rtc/src/redux/actions'
```

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->
Add package `react-router-redux` to the devDependencies in package.json

--- a/package.json
+++ b/package.json
@@ -102,6 +102,7 @@
     "postcss-modules": "^1.3.2",
     "react-dev-utils": "^5.0.1",
     "react-hot-loader": "^4.3.4",
+    "react-router-redux": "^4.0.8",
     "sass-loader": "^7.1.0",
     "strip-json-comments": "^2.0.1",
     "style-loader": "^0.21.0",

